### PR TITLE
feature: add multi_equals

### PR DESCRIPTION
**Pull request by TomBurdge**
This is maybe a bit jank...
I was starting to think I am missing something about the polars expression api while doing this...

### CODE REVIEW

File: tests/test_column_functions.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import pytest
+from tests.conftest import polars_frames
+from harley.column_functions import multi_equals
+from polars import DataFrame, LazyFrame, DataType
+from polars.testing import assert_frame_equal
+
+@pytest.mark.parametrize("frame_type", polars_frames)
+def test_multi_equals(frame_type:DataType):
+    data = frame_type({"col_1": ["cat","dog","pig"],"col_2":["cat","dog","pig"]})
+    exp = DataFrame({"res":[True, False, False]})
```
Comment (LyndonFan) [inline tests/test_column_functions.py]:
```suggestion
    data = frame_type([{"col_1": "cat", "col_2": "cat"}, {"col_1": "cat", "col_2": "dog"}, {"col_1": "dog", "col_2": "cat"}, {"col_1": "dog", "col_2": "dog"}])
    exp = DataFrame({"res": [True, False, False, False]})
```

File: harley/column_functions.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from polars import Expr
+from typing import Any, List
+import polars as pl
+
+def multi_equals(cols: List[str], val: Any) -> Expr:
+    query = [pl.col(name) == val for name in cols]
```
Comment (LyndonFan) [inline harley/column_functions.py:6]: Do we want to warn / handle if `val = None`?

Comment (TomBurdge) [reply harley/column_functions.py:6]: what would the warning be, that some kind of `is_null` equivalent would be better?

Comment (LyndonFan) [reply harley/column_functions.py:6]: Yeah, but to be fair polars does it so maybe just leave it as is

**Pull request merged**